src/components/Resource/Files.tsx

```diff
@@ -289,7 +289,13 @@ function TrackedFileStatus({
                 sx={{ flex: 1 }}
               />
               <Tooltip label="Cancel upload" position="left">
-                <ActionIcon color="red" onClick={() => abort(uuid)}>
+                <ActionIcon
+                  color="red"
+                  onClick={() => {
+                    abort(uuid);
+                    updateFile(versionFileUuid, { status: 'aborted' });
```

Comment:
This works, but I think we should handle this in the `s3-upload.store.ts` file. I made some changes to `useCFImageUpload.ts` to handle updating the file on abort that you could use as an example.

Reply:
After discussing, the solution we got at the moment works. If we need to review it later we can rework it